### PR TITLE
support for WITHSCORES in zrevrange

### PR DESCRIPTION
**Pull request by KashiMoonactive**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,7 @@
   "license": "MIT",
   "author": "Stian Didriksen <stipsan@gmail.com>",
   "homepage": "https://github.com/stipsan/ioredis-mock#readme",
-  "repository": {
```
[the remark this answers is not in the document]
Comment (KashiMoonactive) [reply package.json]: done

**Comment by KashiMoonactive** [comment]
updated my PR following your comments

**Comment by nschirmer** [comment]
This is great, I actually had a need for this today, thank you @KashiMoonactive.

Actually, I have a need for `zrange` to support WITHSCORES, but I can work with `zrevrange` and just reverse the data.

I can get a PR up for `zrange` support of WITHSCORES if @KashiMoonactive or anyone else doesn't beat me to it. :)

**Pull request merged**